swarm/src/lib.rs

```diff
@@ -603,38 +607,56 @@ where
                 Poll::Pending => network_not_ready = true,
                 Poll::Ready(NetworkEvent::ConnectionEvent { connection, event }) => {
                     let peer = connection.peer_id();
-                    let connection = connection.id();
-                    this.behaviour.inject_event(peer, connection, event);
+                    let conn_id = connection.id();
+                    if !this.banned_peer_connections.contains(&conn_id)
+                        || this.banned_peers.contains(&peer)
+                    {
+                        this.behaviour.inject_event(peer, conn_id, event);
+                    } else {
+                        log::debug!(
+                            "Ignoring event from disallowed connection or peer: {:?} {:?}.",
```

Comment:
```suggestion
                            "Ignoring event from banned peer: {:?} {:?}.",
```

Given that a user _banns_ a peer, wouldn't that be more intuitive here?